pkg/crc/preflight/preflight_checks_linux.go

```diff
@@ -123,6 +124,17 @@ func fixLibvirtInstalled() (bool, error) {
 	return true, nil
 }
 
+// TODO: needs refactor
+func fixLibvirtInstalledUbuntu() (bool, error) {
+	logging.Debug("Trying to install libvirt")
+	stdOut, stdErr, err := crcos.RunWithPrivilege("apt-get", "install", "-y", "qemu-kvm", "libvirt-daemon", "libvirt-daemon-system")
+	if err != nil {
```

Comment:
There's a `PackageManager` variable defined in LinuxDistribution, but it's not being used (and I'm not sure it should, as the command syntax is going to be different, the package names are also going to be different)

Reply:
This was for possible solution we might were able to give for a known distro when we havent added all steps. such `Missing ... Use your the package manager %s to install these missing ... blah bla`, just didnt come to this (yet). However, we will solve this differently now.

Reply:
@cfergeau not mentioning the possible packagemanager leads people to use `yum`. This even happened in: https://github.com/code-ready/crc/issues/378#issue-476863030